### PR TITLE
Travis: Test huge pages and disabling Fortran

### PR DESCRIPTION
**Pull request by jdinan**

### CODE REVIEW

**Comment by jdinan** [comment]
Closes #187

**Comment by jdinan** [comment]
Ready for review.

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,8 @@ env:
         - # Defaults
         - SOS_BUILD_OPTS="--disable-mr-scalable --enable-av-map --disable-cxx"
         - SOS_BUILD_OPTS="--enable-cma --enable-error-checking --enable-profiling --disable-mr-scalable --enable-av-map --enable-remote-virtual-addressing"
-        - SMA_BOUNCE_SIZE=0 SOS_BUILD_OPTS="--enable-error-checking --enable-remote-virtual-addressing"
+        - SOS_DISABLE_FORTRAN=1 SOS_BUILD_OPTS="--disable-fortran --enable-error-checking --enable-remote-virtual-addressing"
+        - SMA_SYMMETRIC_HEAP_USE_HUGE_PAGES=1 SMA_BOUNCE_SIZE=0 SOS_BUILD_OPTS="--enable-error-checking --enable-remote-virtual-addressing"
```
Comment (kseager) [inline .travis.yml]: you meant enabling large pages right (referencing the title)? but yes its nice to add this option

File: .travis.yml [hunk 2]
```diff
@@ -111,27 +112,29 @@ script:
     - export PATH=$TRAVIS_INSTALL/portals4/bin:$TRAVIS_INSTALL/sandia-shmem-portals4/bin:$TRAVIS_INSTALL/hydra/bin:$BASE_PATH
     - export OSHRUN_LAUNCHER="mpiexec.hydra"
     - cd $TRAVIS_SRC/tests-uh
-    - make $TRAVIS_PAR_MAKE C_feature_tests F_feature_tests
+    - make $TRAVIS_PAR_MAKE C_feature_tests
+    - test $SOS_DISABLE_FORTRAN || make F_feature_tests
```
Comment (kseager) [inline .travis.yml]: sure this works....

**Comment by kseager** [comment]
looks good to me 👍

**Comment by jdinan** [comment]
Sorry for churning this PR so many times.  The end functionality is still the same, but I cleaned up the YAML and shell scripting in a way that's a bit to maintain.  Once the tests pass, I'll go ahead and merge it in.

**Pull request merged**